ext/afform/core/ang/af/afField.component.js

```diff
@@ -142,8 +142,33 @@
         });
       };
 
+      // correct the type for the value, make sure numbers are numbers and not string
+      function correctValueType(value, dataType) {
+        // let's skip type correction for null values
+        if (value == null) {
```

Comment:
Need to check data type using `===`
```suggestion
        if (value === null) {
```

Reply:
done